### ISSUE TITLE
AdMob 7.4.1 and AppLovin 6.5.1 not working on iOS

### ISSUE THREAD

**Issue by Nyankoo** [googleads/googleads-mobile-unity#2560]
* Unity version: 2021.3.16f1
  * Google Mobile Ads Unity plugin version: 7.4.1
  * Platform: iOS
  * Mediation ad networks used, and their versions: AppLovin 6.5.1

```
[!] CocoaPods could not find compatible versions for pod "Google-Mobile-Ads-SDK":
In Podfile:
Google-Mobile-Ads-SDK (~> 10.0)
GoogleMobileAdsMediationAppLovin (= 11.6.1.0) was resolved to 11.6.1.0, which depends on
Google-Mobile-Ads-SDK (~> 9.8)
Specs satisfying the `Google-Mobile-Ads-SDK (~> 10.0), Google-Mobile-Ads-SDK (~> 9.8)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by NVentimiglia** [comment]
Hi @Nyankoo,

We are aware of this issue and will be prioritizing a fix. Thank you for your patience.

**Comment by dcosmin-97** [comment]
Is it working on Android?

**Comment by programmermattsrum** [comment]
@NVentimiglia Assuming this is the same issue with the ironsrc integration? 
IronSourceUnityAdapter-2.2.7.zip not available yet ??

[!] CocoaPods could not find compatible versions for pod "Google-Mobile-Ads-SDK":
In Podfile:
Google-Mobile-Ads-SDK (~> 10.0)
GoogleMobileAdsMediationIronSource (= 7.2.6.0.0) was resolved to 7.2.6.0.0, which depends on
Google-Mobile-Ads-SDK (~> 9.8)
Specs satisfying the `Google-Mobile-Ads-SDK (~> 10.0), Google-Mobile-Ads-SDK (~> 9.8)` dependency were found, but they required a higher minimum deployment target.

**Comment by oddgames-david** [comment]
I assume it's most of the mediation adapters?
```
[!] CocoaPods could not find compatible versions for pod "Google-Mobile-Ads-SDK":

  In Podfile:

    Google-Mobile-Ads-SDK (~> 10.0)



    GoogleMobileAdsMediationAdColony (= 4.9.0.0) was resolved to 4.9.0.0, which depends on

      Google-Mobile-Ads-SDK (~> 9.0)



Specs satisfying the `Google-Mobile-Ads-SDK (~> 10.0), Google-Mobile-Ads-SDK (~> 9.0)` dependency were found, but they required a higher minimum deployment target.


```

**Comment by programmermattsrum** [comment]
> 

@NVentimiglia @vinkini  I see the changes were merged?  Is this available in the current release or will there be another release pushed?

**Comment by oddgames-david** [comment]
> > 
> 
> @NVentimiglia @vinkini I see the changes were merged? Is this available in the current release or will there be another release pushed?

Can we get some response to this? Waiting for a new version so we can update for iOS

**Comment by psrar** [comment]
Same for Unity Ads adapter

**Comment by NVentimiglia** [comment]
@programmermattsrum @oddgames-david 

This will be in the next version release. We are aiming for this week. Thank you for your patience.

**Comment by takazawa-gg** [comment]
@NVentimiglia 
I am keeping an eye on the adapter updates on Git, but it appears that Maio's updates are not progressing.
I am hoping that a Maio adapter will be created for plugin version: 7.4.1.

**Comment by NVentimiglia** [comment]
Fixed in the latest release (8.0.0).

**Comment by xy2080** [comment]
> 已在最新版本 (8.0.0) 中修复。

Unity 2020.3.21f1
